src/node-fs.js

```diff
@@ -22,5 +22,8 @@ module.exports = {
       return fs.readFileSync(url.fileURLToPath(path), encoding)
     }
     return fs.readFileSync(path, encoding)
+  },
+  dirname: (resourceId) => {
+    return resourceId.substring(0, Math.max(resourceId.lastIndexOf('/'), resourceId.lastIndexOf('\\')) + 1)
```

Comment:
we can use `path.dirname`: https://nodejs.org/api/path.html#pathdirnamepath

Reply:
I was thinking that, but it depends on what happens to that resourceId before it reaches that function. In any case, it still didn't fix the issue on the Windows build. I'll need to set up a Windows environment to test and debug it locally.

Reply:
In theory, `path.dirname` should use `path.sep` which is OS depend so it should work fine on Linux or Windows.